src/main/java/io/zeebe/containers/ZeebeBrokerNode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2019 camunda services GmbH (info@camunda.com)
```

Comment:
The header is pulled from the parent project, so that would have to be done there, no?

Reply:
Ah, I was unaware of that. Interestingly, this project uses `org.camunda:camunda-release-parent:3.7` instead of `io.zeebe:zeebe-parent:0.24.0`. Zeebe's header does not have a year mentioned.

Reply:
zeebe-parent is unfortunately not really meant to be used as a generic parent for any module. It's really tailored for zeebe modules, which is why you'll find most community projects even for Zeebe use the camunda-release-parent as a parent.